client/src/pages/EditEntry.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import UniversalButton from '../components/UniversalButton';
+import { useParams, useHistory } from 'react-router-dom';
+import PageWrapperCenterSpEvenly from '../components/PageWrapperCenterSpEvenly';
+
+const Textarea = styled.textarea`
+  width: 100%;
+  border: 1px solid white;
+  border-radius: 8px;
+  color: ${props => props.theme.colors.background};
+  background-color: ${props => props.theme.colors.quinary};
+  font-family: 'Roboto', sans-serif;
+`;
+
+const Card = styled.form`
+  width: 85%;
+  height: 500px;
+  border-radius: 8px;
+  padding: 10px;
+  font-family: 'Roboto', sans-serif;
+  color: ${props => props.theme.colors.primary};
+  background-color: ${props => props.theme.colors.quinary};
+  opacity: 0.75;
+  margin: 15px;
+  overflow: scroll;
+`;
+const ButtonWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+  margin: 4px;
+  justify-content: flex-start;
+  align-items: space-around;
+  margin-top: 8px;
+`;
+
+function EditEntry() {
+  const [edit, setEdit] = React.useState([]);
```

Comment:
You are mixing the type of edit in this file.
The initial value is an empty array. 
In the onChange, you change it to an object. 
In the fetch PATCH, you use it to set the name.

Reply:
Changed this after refactoring, thank you.